apps/test/unit/templates/sectionProgress/progressTables/ProgressTableContainerTest.jsx

```diff
@@ -0,0 +1,151 @@
+import {UnconnectedProgressTableContainer as ProgressTableContainer} from '@cdo/apps/templates/sectionProgress/progressTables/ProgressTableContainer';
+import ProgressTableStudentList from '@cdo/apps/templates/sectionProgress/progressTables/ProgressTableStudentList';
+import ProgressTableContentView from '@cdo/apps/templates/sectionProgress/progressTables/ProgressTableContentView';
+
+const LESSON_1 = {
+  name: 'CS Discoveries Pre-survey',
+  relative_position: 1,
+  lockable: false,
+  position: 1,
+  levels: [
+    {
+      id: 22484,
+      url: '/s/csd1-2020/lockable/1/puzzle/1/page/1',
+      kind: 'assessment',
+      icon: 'fa fa-list-ul',
+      isUnplugged: false,
+      levelNumber: 1,
+      bubbleText: '1',
+      isConceptLevel: false,
+      bonus: null
+    }
+  ]
+};
+
+const LESSON_2 = {
+  name: 'Intro to Problem Solving',
+  relative_position: 2,
+  lockable: false,
+  position: 2,
+  levels: [
+    {
+      id: 12345,
+      url: '/s/csd1-2020/lockable/1/puzzle/1/page/1',
+      kind: 'assessment',
+      icon: 'fa fa-list-ul',
+      isUnplugged: false,
+      levelNumber: 1,
+      bubbleText: '1',
+      isConceptLevel: false,
+      bonus: null
+    }
+  ]
+};
+
+const DEFAULT_PROPS = {
+  onClickLesson: () => {},
+  getTableWidth: () => 500,
+  columnWidths: [50, 100, 75, 50],
+  lessonCellFormatter: () => {},
+  extraHeaderFormatters: [],
+  extraHeaderLabels: [],
+  children: <div />,
+  section: {id: 1, students: [{id: 1, name: 'joey'}, {id: 2, name: 'rachael'}]},
+  scriptData: {
+    id: 1,
+    name: 'csd1-2020',
+    title: 'CSD Unit 1 - Problem Solving and Computing (20-21)',
+    stages: [LESSON_1, LESSON_2]
+  },
+  lessonOfInterest: 1,
+  levelProgressByStudent: {
+    1: {
+      1: {
+        status: 'perfect',
+        result: 100,
+        paired: false
+      }
+    },
+    2: {
+      1: {
+        status: 'perfect',
+        result: 100,
+        paired: false
+      }
+    }
+  },
+  studentTimestamps: {1: 0, 2: 0},
+  localeCode: 'en-US'
+};
+
+const setUp = (overrideProps = {}) => {
+  const props = {...DEFAULT_PROPS, ...overrideProps};
+  return mount(<ProgressTableContainer {...props} />);
+};
+
+describe('ProgressTableContainer', () => {
+  it('renders a ProgressTableStudentList', () => {
+    const wrapper = setUp();
+    expect(wrapper.find(ProgressTableStudentList)).to.have.length(1);
+  });
+
+  it('renders a ProgressTableContentView', () => {
+    const wrapper = setUp();
+    expect(wrapper.find(ProgressTableContentView)).to.have.length(1);
+  });
+
+  it('passes needsGutter true to the ProgressTableContentView when the student row height exceeds the body height', () => {
+    // 18 students will exceed max height
+    const students = Array(18)
+      .fill()
+      .map((x, i) => ({id: i, name: `student-${i}`}));
+
+    const wrapper = setUp({section: {id: 1, students: students}});
+    expect(wrapper.find(ProgressTableContentView).props().needsGutter).to.equal(
```

Comment:
I think `to.be.true` is the more canonical way to check this condition?  Similar comments for other boolean checks in this file.  (I'm not sure though, I'm just learning chai too.)

Reply:
Yup, you're right, I'll update those statements!